_overviews/scala3-book/scala-features.md

```diff
@@ -128,15 +128,13 @@ As Heather Miller states, Scala is considered to be a [strong, statically-typed
 
 - Correctness: you catch most errors at compile-time
 - Great IDE support
-    - Code completion
+    - Reliable code completion
     - Catching errors at compile-time means catching mistakes as you type
     - Easy and reliable refactoring
-    - Reliable code completion
```

Comment:
I found that “code completion” and “reliable code completion” were too redundant, so I simplified to just “reliable code completion”

Reply:
Thanks! I didn’t see that duplication.